src/_P033_Dummy.ino

```diff
@@ -98,6 +98,37 @@ boolean Plugin_033(byte function, struct EventStruct *event, String& string)
         success = true;
         break;
       }
+
+    case PLUGIN_WRITE:
+      {
+        String command = parseString(string, 1);
+        if (command == F("dummyvalueset"))
+        {
+          int deviceNr=parseString(string, 2).toInt(); // Index from 1-12;
+          if (deviceNr == event->TaskIndex+1) // make shure that this instance is the target
+          {
+            int valueNr=parseString(string, 3).toInt();
+            float value=0;
+            String valueStr=parseString(string, 4);
+            if (valueStr=="true") value=1;
+              else if(valueStr=="false") value=0;
+              else value=valueStr.toFloat();
```

Comment:
Please use { } to mark the scope of an if-statement.
Especially when there is an else scope and even more in this case with if...else if...else.

Also use `F()` macro for strings, or else they will be placed in RAM.

And there is also a function to check if it is a valid float.

Reply:
DummyValueSet will never receive true of false so I erased that part.
Hope I found all missing {} and added all F() macros.
Use string2float() instead .toFloat